src/app/Fake.FluentMigrator/FluentMigrator.fs

```diff
@@ -0,0 +1,311 @@
+﻿/// Contains functions to run FluentMigrator
+module Fake.FluentMigrator
```

Comment:
Maybe `Fake.Sql.FluentMigrator` or `Fake.DotNet.FluentMigrator`?

Reply:
+`RequireQualifiedAccess` and a sample in the doc string would be nice, see
https://github.com/fsharp/FAKE/blob/83f79896d303c081039fd887885ca5c74c95526b/src/app/Fake.Core.ReleaseNotes/ReleaseNotes.fs#L1-L39

Reply:
I can put it under `Fake.Sql.FluentMigrator` 
It's not really specific to .NET Core is it? `Sql` seems more appropriate

Reply:
Yes I wasn't sure about that (as I don't really know the tool) so Fake.Sql is fine, we already have something there